basxconnect/core/views/menu_views.py

```diff
@@ -49,3 +52,33 @@
         settingsgroup,
     )
 )
+
+menu.registeritem(
+    menu.Item(
+        Link(
+            reverse("bread.views.system.systeminformation"),
```

Comment:
The URL which we registered inside bread (as mentioned above) is "systeminformation", so this will no longer work. By the way: You can always use ```python manage.py show_urls``` to get the name of a URL.